src/components/notifications/Discord.vue

```diff
@@ -16,4 +16,36 @@
         <label for="discord-prefix-message" class="form-label">{{ $t("Prefix Custom Message") }}</label>
         <input id="discord-prefix-message" v-model="$parent.notification.discordPrefixMessage" type="text" class="form-control" autocomplete="false" :placeholder="$t('Hello @everyone is...')">
     </div>
+
+    <div class="mb-3">
+        <label for="discord-message-type" class="form-label">{{ $t("Select message type") }}</label>
+        <br>
+        <select id="discord-message-type" v-model="$parent.notification.discordChannelType" class="form-select">
+            <option value="normal">{{ $t("Normal Message") }}</option>
```

Comment:
Current behaviour is to post to a channel (?). Might this be a better compared to `Normal Message`